target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-rb3011uias.dts

```diff
@@ -0,0 +1,599 @@
+		status = "okay";
+
+		pinctrl-0 = <&spi_pins>;
+		pinctrl-names = "default";
+
+		cs-gpios = <&qcom_pinmux 20 GPIO_ACTIVE_HIGH>;
+
+		flash: m25p80@0 {
+			compatible = "jedec,spi-nor";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			spi-max-frequency = <51200000>;
+			reg = <0>;
+
+			partitions {
+				compatible = "fixed-partitions";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				SBL1@0 {
+					label = "SBL1";
+					reg = <0x00000000 0x20000>;
+					read-only;
+				};
+				MIBIB@20000 {
+					label = "MIBIB";
+					reg = <0x00020000 0x20000>;
+					read-only;
+				};
+				SBL2@40000 {
+					label = "SBL2";
+					reg = <0x00040000 0x40000>;
+					read-only;
+				};
+				SBL3@80000 {
+					label = "SBL3";
+					reg = <0x00080000 0x80000>;
+					read-only;
+				};
+				DDRCONFIG@100000 {
+					label = "DDRCONFIG";
+					reg = <0x00100000 0x10000>;
+					read-only;
+				};
+				SSD@110000 {
+					label = "SSD";
+					reg = <0x00110000 0x10000>;
+					read-only;
+				};
+				TZ@120000 {
+					label = "TZ";
+					reg = <0x00120000 0x80000>;
+					read-only;
+				};
+				RPM@1a0000 {
+					label = "RPM";
+					reg = <0x001a0000 0x20000>;
+					read-only;
+				};
+				APPSBL@1c0000 {
+					label = "APPSBL";
+					reg = <0x001c0000 0x40000>;
+					read-only;
+				};
+				hard_config: RBHARDCONFIG@1cf000 {
+					label = "hard_config";
+					reg = <0x001cf000 0x01000>;
+					read-only;
+				};
+				RBSOFTCONFIG@1e0000 {
+					label = "soft_config";
+					reg = <0x001e0000 0x01000>;
+				};
+			};
+		};
+	};
+};
+
+&gsbi4 {
+	status = "disabled";
+};
+
+&gsbi7 {
+	qcom,mode = <GSBI_PROT_I2C_UART>;
+	status = "okay";
+};
+
+&gsbi7_serial {
+	status = "okay";
+};
+
+&nand_controller {
+	status = "okay";
+
+	pinctrl-0 = <&nand_pins>;
+	pinctrl-names = "default";
+
+	nand@0 {
+		reg = <0>;
+		compatible = "qcom,nandcs";
+
+		nand-ecc-strength = <4>;
+		nand-bus-width = <8>;
+		nand-ecc-step-size = <512>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "kernel";
+				reg = <0x00000000 0x1000000>;
+			};
+
+			partition@1000000 {
+				label = "ubi";
+				reg = <0x1000000 0x7000000>;
+			};
+		};
+	};
+};
+
+&hs_phy_0 { /* USB3 port 0 HS phy */
+	clocks = <&gcc USB30_1_UTMI_CLK>;
+	status = "okay";
+};
+
+&ss_phy_0 { /* USB3 port 0 SS phy */
+	clocks = <&gcc USB30_1_MASTER_CLK>;
+	status = "okay";
+};
+
+&usb3_0 {
+	clocks = <&gcc USB30_1_MASTER_CLK>;
+	pinctrl-0 = <&usb0_pwr_en_pins>;
+	pinctrl-names = "default";
+	no-clk_bulk_get;
+	no-dwc3_qcom_setup_irq;
+	status = "okay";
+};
+
+&mdio0 {
+	status = "okay";
+
+	pinctrl-0 = <&mdio0_pins &sw0_and_phy_reset_pin>;
+	pinctrl-names = "default";
+
+	sw0_phy_port1: sw0-phy@0 {
+		reg = <0>;
+	};
+	sw0_phy_port2: sw0-phy@1 {
+		reg = <1>;
+	};
+	sw0_phy_port3: sw0-phy@2 {
+		reg = <2>;
+	};
+	sw0_phy_port4: sw0-phy@3 {
+		reg = <3>;
+	};
+	sw0_phy_port5: sw0-phy@4 {
+		reg = <4>;
+	};
+
+	switch0: switch@10 {
```

Comment:
This label is not used, same appears for switch1 and several others. Please remove them all.